recipe/meta.yaml

```diff
@@ -1,33 +1,42 @@
+{% set name = "jupyter_server_terminals" %}
 {% set version = "0.4.4" %}
+# there is a test dependency on jupyter_server, but jupyter_server has
+# a runtime dependency on this package. Build first with true, then
+# after jupyter_server is available bump build number and rebuild with false.
+{% set break_cycle = true %}
 
 package:
-  name: jupyter_server_terminals
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/j/jupyter-server-terminals/jupyter_server_terminals-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: 57ab779797c25a7ba68e97bcfb5d7740f2b5e8a83b5e8102b10438041a7eac5d
 
 build:
-  script: {{ PYTHON }} -m pip install . -vv
-  number: 1
-  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  number: 0
+  skip: true # [py<38]
 
 requirements:
   host:
     - hatchling >=1.5
     - pip
-    - python >=3.8
+    - python
+    - wheel
   run:
-    - python >=3.8
+    - python
     - terminado >=0.8.3
     # - pywinpty >=2.0.3  # rely on terminado to provide this
```

Comment:
`terminado` has a non-compatible pinning for `pywinpty`, see https://github.com/AnacondaRecipes/terminado-feedstock/blob/c33101b98039f53ecfda9124c2e54a7f6c419477/recipe/meta.yaml#L28
I would recommend enabling `- pywinpty >=2.0.3  # [win]`.